app/models/article.rb

```diff
@@ -1,4 +1,15 @@
 class Article < ApplicationRecord
   acts_as_taggable_on :tags
   validates :title, :date, :author, :content, :image, presence: true
+
+  include PgSearch::Model
+  pg_search_scope :search_by_title_content_and_author,
+                  against: {
+                    title: 'A',
```

Comment:
maybe content could be first? and perhaps lets remove author as we won't have any unfortunately ...

Reply:
Sounds good, will change it up